cmd/mr_approve.go

```diff
@@ -52,6 +54,11 @@ var mrApproveCmd = &cobra.Command{
 
 		err = lab.MRApprove(p.ID, int(id))
 		if err != nil {
+			if strings.Contains(err.Error(), "{message: 403 Forbidden}") {
```

Comment:
Instead of comparing to the error string, wouldn't it be better to check the "resp" return value in `internal/gitlab/gitlab.go:MRApprove()` returned by `lab.MergeRequestApprovals.ApproveMergeRequest()` and create a new error type as it's already done for both `ErrProjectNotFound` and `ErrGroupNotFound` ?

Reply:
Additionally, `http.StatusForbidden` could be used in comparison with the `resp` return value.

Reply:
Oh, I had no idea I could do it that way.  Let me look into it!

Reply:
Updated :)  Thanks for the suggestion @bmeneguele !

Reply:
It seems you dropped the `cmd/mr_{un}approve.go` changes, the ones actually checking the error and printing better logs :)